pkg/controller/appdefinition/volume.go

```diff
@@ -155,7 +156,31 @@ func toPVCs(req router.Request, appInstance *v1.AppInstance) (result []kclient.O
 
 		if bind {
 			pvc.Name = bindName(vol)
-			pvc.Spec.VolumeName = volumeBinding.Volume
+
```

Comment:
I've thought through the below logic in the context of whether or not you could use this to "steal" another unbound volume from another acorn project. Looks like the
```
				labels.AcornAppNamespace: appInstance.Namespace,
```
bit of the selector prevents that. Is that right?

Reply:
Yeah, I think with the way things are right now you can steal an unbound PV from another project. My changes should prevent that from being possible.